packages/emails/email-manager.ts

```diff
@@ -27,45 +27,32 @@ import OrganizerScheduledEmail from "./templates/organizer-scheduled-email";
 import type { TeamInvite } from "./templates/team-invite-email";
 import TeamInviteEmail from "./templates/team-invite-email";
 
+// eslint-disable-next-line
+const sendEmail = (prepare: () => any) => {
```

Comment:
```suggestion
const sendEmail = (prepare: () => BaseEmail) => {
```

This seems to compile correctly.